microsoft/testsuites/performance/storageperf.py

```diff
@@ -102,12 +102,36 @@ def perf_premium_datadisks_1024k(
             test_case_name="perf_premium_datadisks_1024k",
         )
 
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test vm with 24 data disks.
+        """,
+        priority=3,
+        timeout=TIME_OUT,
+        requirement=simple_requirement(
+            min_core_count=72,
+            disk=schema.DiskOptionSettings(
+                disk_type=schema.DiskType.PremiumSSDLRS,
+                data_disk_iops=5000,
```

Comment:
set the iops as min value, so it can be more flexible.

Reply:
it is to test iops of P30 disk, it has the minimal requirement for the iops.

Reply:
Set it to a min value, not a number directly.

Reply:
Updated. @squirrelsc @LiliDeng  Should we set a min value in other storage performance test cases too?